ppb/camera.py

```diff
@@ -0,0 +1,67 @@
+from ppb import Vector
+from ppb import BaseSprite
+from ppb.flags import DoNotRender
+
+class Camera(BaseSprite):
+
+    image = DoNotRender
+
+    def __init__(self, viewport=(0, 0, 800, 600), pixel_ratio=80):
+        """
+
+        viewport: A container of origin x, origin y, width, and
+                  height in pixel space.
+        pixel_ratio: An number defining the pixel to game space ratio.
+                     Effectively a "zoom".
+        """
```

Comment:
Soooo.... viewport width/height divided by the pixel ratio is the actual window width/height in screen pixels? It should probably not be labeled "pixel space".

Also, which corner is the origin again?

Reply:
Opposite: Viewport width x height == actual pixel width and height of the viewport. Divide by pixel ratio to get to game space.

Top left is the origin. Definitely should not that.